### ISSUE TITLE
Pushnotification when interesting pokemon spawn

### ISSUE THREAD

**Issue by Timvdv** [TheZ3ro/pkmngo-map#27]
Hey guys,

I just made a few changes to this script to send push notification via pushbullet when a interesting Pokemon spawns.

The code is really dirty atm.. but it works. If there is more interest in this I can clean it up a bit and push it to this repo

**Comment by TheZ3ro** [comment]
You can push the code and it will be reviewed

**Comment by Timvdv** [comment]
This is the first time I wrote something in Python, maybe you can help me out. I was having some trouble with Pokemon that were logged multiple times (every new scan the same Pokemon are logged. Not talking about doubles in the same scan here). So I came up with a solution. When a push notification is send the Pokemon is added to the list with its name, lat and lng. Before it sends the push it checks if the Pokemon is added to the list. Now I want to clear the list every hour because there are Pokemon that can spawn on the same location.

I used the following code to clear the list every hour(loglist is defined at the top):
```
def clearLogList():
    del loglist[:]
    threading.Timer(3600, clearLogList).start()

clearLogList();
```

The problem is.. I can't close the script anymore. I get the following error message:
```
  File "main.py", line 553, in <module>
    main()
  File "main.py", line 550, in main
    scan(api_endpoint, access_token, response, origin, pokemons)
  File "main.py", line 399, in scan
    hs.append(heartbeat(api_endpoint, access_token, response))
  File "main.py", line 371, in heartbeat
    h = raw_heartbeat(api_endpoint, access_token, response)
  File "main.py", line 359, in raw_heartbeat
    m5)
  File "main.py", line 272, in get_profile
    return api_req(api, access_token, req, useauth = useauth)
  File "main.py", line 235, in api_req
    time.sleep(0.51)
KeyboardInterrupt
```

I have to close the terminal to restart the script.. This is pretty annoying. I think this is because of the new thread I implemented but I'm to inexperienced with Python to know for sure.

**Comment by TheZ3ro** [comment]
Yes it's because the new thread, every thread must end its execution for exit the program. Using thread in an application like this is a lot of pain.

You can use a counter variable that you increment by 1 in the scan function and when it's like 100 you send the request

**Comment by Timvdv** [comment]
Awesome, working pretty smoothly now. I'm working on one more feature, when this works I'll create a pull request

**Comment by TheZ3ro** [comment]
Please, make 1 pull request for every feature so it's easier for me to understand your code :wink:
